tests/association/test_positioning.py

```diff
@@ -92,6 +92,29 @@ def test_positioned_above_same_page_false(input, expected):
     is_above = positioning.positioned_above(input[0], input[1], must_be_same_page=False)
     assert is_above == expected
 
+@pytest.mark.parametrize(
+    "input, expected",
+    [
+        ((generate_mapped_pred(), generate_mapped_pred(11, 20, 10, 20)), False),
+        ((generate_mapped_pred(), generate_mapped_pred(11, 20, 6, 15)), False),
+        ((generate_mapped_pred(), generate_mapped_pred(11, 20, 1, 9)), True),
+        ((generate_mapped_pred(), generate_mapped_pred(11, 20, 4, 15)), True),
+    ]
```

Comment:
might want to add one more test case that validates that you don't have any directional bugs- i.e. top box is currently 0l 10r, only check fully underneath and left side of lower box overlaps- move upper box to the right have overlap with the right side of the lower box